tests/Adapter/WorseReflection/Refactor/WorseGenerateMethodTest.php

```diff
@@ -89,11 +89,17 @@ public function greet()
         $this->generateMethod($source, 152, 'test_name');
     }
 
-    private function generateMethod(string $source, int $start, $name)
+    private function generateMethod(string $source, int $start, ?string $name): string
     {
         $reflector = $this->reflectorForWorkspace($source);
         $factory = new WorseBuilderFactory($reflector);
         $generateMethod = new WorseGenerateMethod($reflector, $factory, $this->updater());
-        return $generateMethod->generateMethod(SourceCode::fromString($source), $start, $name);
+        $sourceCode = SourceCode::fromString($source);
+        $edits = $generateMethod->generateMethod($sourceCode, $start, $name);
+        $transformed = SourceCode::fromStringAndPath(
+            (string) $edits->apply($sourceCode),
+            $sourceCode->path()
+        );
+        return $transformed;
```

Comment:
not important, but could inline some of this and remove the temporary variables.